services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProviderConfig.java

```diff
@@ -52,6 +52,8 @@
     public static final String WANT_ASSERTIONS_SIGNED = "wantAssertionsSigned";
     public static final String WANT_AUTHN_REQUESTS_SIGNED = "wantAuthnRequestsSigned";
     public static final String XML_SIG_KEY_INFO_KEY_NAME_TRANSFORMER = "xmlSigKeyInfoKeyNameTransformer";
+    public static final String HIDE_ON_LOGIN = "hideOnLoginPage";
+    public static final String ENABLED = "enabled";
```

Comment:
Remove, there is already `enabled` property in the parent class.

Reply:
I know it. The problem is that importFrom method in class IdentityProvidersResource ( rest api for import identity provider from url or file) is returning only IdentityProviderModel.config. Property enabled in IdentityProviderModel is not in this map. As a result when you import a Saml Identity Provider from url or file , property enabled will not returned from rest api. Do you suggest another solution?

Reply:
In that case it is necessary to use a specific name, plain `enable` would be confusing with the property above.
```suggestion
    public static final String ENABLED_FROM_METADATA = "enabledFromMetadata";
```